docs/integrations/webhooks/sync-data.mdx

```diff
@@ -347,33 +281,34 @@ This guide can be adapted to listen for any Clerk event.
 
-  #### Message failed
+  #### Handling failed messages
 
-  If the message failed:
-
-  1. Select the message.
+  1. In the **Message Attempts** section, select the event labeled with `Failed`.
   1. Scroll down to the **Webhook Attempts** section.
-  1. Select the arrow icon the left side.
-  1. Investigate the error. Your solution is going to depend on the error message. See the [Debug your webhooks](/docs/integrations/webhooks/debug-your-webhooks) guide for more information.
+  1. Toggle the arrow next to the **Status** column.
+  1. Review the error. Solutions vary by error type. For more information, refer to the [Debug your webhooks](/docs/integrations/webhooks/debug-your-webhooks) guide.
 
-  ### Trigger your webhook
+  ### Trigger the webhook
 
-  To trigger the `user.updated` event, you can do one of the following:
+  To trigger the `user.updated` event, you can do either one of the following:
 
-  1. You can edit your user in the Dashboard.
-  1. You can use the `<UserProfile />` component in your application to edit your profile.
+  1. Edit your user in the Clerk Dashboard.
+  1. Select the `<UserProfile />` component in your app to edit your profile.
 
-  Once you have updated a user, you should be able to see the webhook's payload logged to your terminal. You can also check the Clerk Dashboard to see the webhook attempt, the same way you did when [testing the webhook](#test-your-webhook).
+  Once you have updated a user, you should be able to see the webhook's payload logged to your terminal. You can also check the Clerk Dashboard to see the webhook attempt, the same way you did when [testing the webhook](#test-the-webhook).
+</Steps>
 
-  ### Wrap up
+### Configure your production instance
 
-  In this guide, you learned how to create a Clerk webhook using Svix. You created a webhook in the Clerk Dashboard to listen for the `user.updated` event and created a Route Handler for your webhook endpoint to verify the webhook signature and receive the payload. You tested the webhook locally using ngrok and the Clerk Dashboard to ensure it was configured properly. And lastly, you added logic to your application to actually trigger the `user.updated` event. Now, you can do whatever you want with the webhook payload, such as sending a notification to your users, updating a database, or anything else you can think of.
-</Steps>
+1. When you're ready to deploy your app to production, follow [the guide on deploying your Clerk app to production](/docs/deployments/overview).
+1. Create your production webhook by following the steps in the previous [Set up a webhook endpoint](#set-up-a-webhook-endpoint) section. In the **Endpoint URL** field, instead of pasting the ngrok URL, paste your production app URL.
+1. After you've set up your webhook endpoint, you'll be redirected to your endpoint's settings page. Copy the **Signing Secret** and update your `.env.local` file with the new **Signing Secret**.
```

Comment:
```suggestion
1. After you've set up your webhook endpoint, you'll be redirected to your endpoint's settings page. Copy the **Signing Secret**.
```

Don't update `.env.local` with the production Signing Secret. That would only go in the hosting platform. `.env.local` would retain the initial/development Signing Secret.